### ISSUE TITLE
"Choose a reward" reward bags don't hide their tier like in older versions

### ISSUE THREAD

**Issue by vizthex123** [lorddusk/HQM#536]
Title. It was nice to be able to surprise someone with the rewards being different tiers, but now it's not possible since the entire tooltip is displayed.

Could this be fixed, or added as a config option?

**Comment by kirderf1** [comment]
Yeah, looks like it was removed with 8dc5b5d2e44eefcaaf6585623431f37544d89c98. My suggestion is an item tag that determines which items will have their description hidden when rendered as a reward. An alternative suggestion is a config option which when enabled will hide specifically reward bag descriptions.
The default should be to always render the description.
